deeppavlov/core/data/simple_vocab.py

```diff
@@ -31,21 +31,21 @@
 class SimpleVocabulary(Estimator):
     """Implements simple vocabulary."""
     def __init__(self,
-                 special_tokens=tuple(),
-                 default_token=None,
-                 max_tokens=2**30,
-                 min_freq=0,
-                 pad_with_zeros=False,
-                 unk_token=None,
+                 special_tokens: tuple = tuple(),
```

Comment:
```suggestion
                 special_tokens: Tuple[str, ...] = tuple(),
```
https://docs.python.org/3/library/typing.html#typing.Tuple